packages/analytics-js/src/components/eventRepository/constants.ts

```diff
@@ -0,0 +1,9 @@
+const DEFAULT_QUEUE_OPTIONS = {
```

Comment:
probably this one also belongs to the  XhrPlugin plugin

Reply:
Addressed.